CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Cromwell Change Log
 
+## 80 Release Notes
+
+### Direct WES support in Cromwell
+
+Cromwell 80 will no longer support the wes2cromwell project within the Cromwell repository.
+
+In the previous release, 3 Wes2Cromwell endpoints in the Cromwell project were implemented and documented in the Swagger API. Three new endpoints,
+located within the wes2cromwell project, will also be moved, implemented, and documented within Cromwell. As a result of this, we can safely remove 
+and deprecate the wes2cromwell project from the repo.
```

Comment:
I really liked the "WES endpoints preview" table in the 79 release notes, I think it might be more readable to expand that format to 6 lines than attempt everything in prose. (It's not clear what the 3 endpoints are in each case.)